atomicapp/cli/main.py

```diff
@@ -540,5 +540,6 @@ def main():
     cli = CLI()
     cli.run()
 
+
```

Comment:
@cdrage flake8 was updated last week and since the version is not locked down for CI testing it failed because the latest version requires a double newline between all def statements and the start of the program. Adding this newline passed the CI tests.